app/controllers/search_controller.rb

```diff
@@ -13,10 +13,13 @@ def extract_filters
 
   def log_metrics_event
     if params[:type] && params[:data] then
-      if params[:other_data] && params[:other_data] != "" then
-        metrics_event(params[:type], params[:data], eval(params[:other_data])) 
-      else 
-        metrics_event(params[:type], params[:data])
+      if params[:type] == 'reverb_redirect'
+        if params[:other_data] && params[:other_data] != "" then
```

Comment:
like what you did for ```:type```, do the same thing for ```:other_data```; find out all the possible values and whitelist them.